lib/pmdtester/parsers/options.rb

```diff
@@ -0,0 +1,55 @@
+require 'slop'
+
+module PmdTester
+  class Options
+
+    VERSION = '1.0.0-SNAPSHOP'
```

Comment:
SNAPSHOP?

Reply:
I'm not sure about it. Please see [comment](https://github.com/pmd/pmd-regression-tester/pull/1#discussion_r187400561).